app/main.py

```diff
@@ -1 +1,31 @@
-# write your code here
+import shlex
+import os
+
+
+def move_file(command: str) -> None:
+    if command[:3] != "mv ":
+        raise ValueError(
+            "Expected 'mv' as the first argument"
+        )
+
+    source_path, destination_path, *extra_path = shlex.split(command[3:])
+    destination_dir, destination_file = os.path.split(destination_path)
+
+    if extra_path:
+        raise ValueError(
+            "Expected 2 arguements"
+        )
+
+    if not os.path.exists(source_path):
+        raise FileExistsError(f"File {source_path} not exists")
+
+    if destination_dir and not os.path.exists(destination_dir):
```

Comment:
> Also you anyway don't need to check `not os.path.exists(source_path)` twice. It will raise the error after the first check
If the file is not found, then directories will be created, and to prevent this, I check beforehand